.github/workflows/vast.yaml

```
@@ -1096,3 +1096,43 @@ jobs:
       - name: Run unit tests
         working-directory: python
         run: poetry run pytest
+      - name: Build package
+        working-directory: python
+        run: poetry build
+
+  python-package:
+    needs:
+      - python
+      - determine-version
+    if: github.event_name == 'release'
+    name: Python Package
+    runs-on: ubuntu-latest
+    env:
+      DEBIAN_FRONTEND: noninteractive
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Setup Poetry image
+        uses: abatilo/actions-poetry@v2.1.6
+        with:
+          poetry-version: 1.2.1
+      - name: Configure Test PyPI
+        working-directory: python
+        run: |
+          poetry config repositories.test-pypi https://test.pypi.org/legacy/
+          poetry config pypi-token.test-pypi "${{ secrets.TEST_PYPI_TOKEN }}"
+      - name: Configure PyPI
+        working-directory: python
+        run: |
+          poetry config pypi-token.pypi "${{ secrets.PYPI_TOKEN }}"
+      - name: Publish ${{ needs.determine-version.outputs.release-version }} to Test PyPI
+        working-directory: python
+        run: poetry publish --build -r test-pypi
+      - name: Publish to PyPI
+        working-directory: python
+        run: poetry publish --build
```

Style request for consistency:

```suggestion
        run: |
          poetry publish --build -r test-pypi
      - name: Publish to PyPI
        working-directory: python
        run: |
          poetry publish --build
```